accounts/abi/bind/bind.go

```diff
@@ -439,10 +407,8 @@ func alias(aliases map[string]string, n string) string {
 }
 
 // methodNormalizer is a name transformer that modifies Solidity method names to
-// conform to target language naming conventions.
-var methodNormalizer = map[Lang]func(string) string{
-	LangGo: abi.ToCamelCase,
-}
+// conform to Go naming conventions.
+var methodNormalizer = abi.ToCamelCase
```

Comment:
Why is this needed at all (now that you removed the lang-dependent map). It's never actually changed, is it? So why not just use `abi.ToCamelCase` and keep it simple?

Reply:
Same with `capitalise` below, really